process/smartContract/process.go

```diff
@@ -868,6 +868,11 @@ func (sc *scProcessor) ExecuteBuiltInFunction(
 		}
 	}
 
+	ignorableError := sc.txLogsProcessor.SaveLog(txHash, tx, vmOutput.Logs)
```

Comment:
no need for this - this may duplicate some SaveLog -> delete the code from executeSmartContractCall. and put it inside finishSCExecution

Reply:
done.